app/helpers/people_helper.rb

```diff
@@ -0,0 +1,3 @@
+module PeopleHelper
+  include Pagy::Frontend
+end
```

Comment:
What do you think about a concern that handles both the backend and frontend inclusion, eg:
```ruby
# app/controllers/concerns/pagination.rb
module Pagination
  extend ActiveSupport::Concern

  include Pagy::Frontend
  include Pagy::Backend

  included do
    helper_method :pagy_bulma_nav
  end
end
```

Reply:
Oh that's good, I'll do this.